app/scripts/modules/appengine/helpContents/appengineHelpContents.ts

```diff
@@ -61,6 +69,11 @@ module(APPENGINE_HELP_CONTENTS_REGISTRY, [HELP_CONTENTS_REGISTRY])
                 (Spinnaker load balancer) will be stopped. This option will be respected only if this server group will
                 be receiving all traffic and the previous server group is using manual scaling.`,
       },
+      {
+        key: 'appengine.serverGroup.containerImageUrl',
+        value: `The full URL to the container image to use for deployment. For example,
```

Comment:
I might mention the scope of allowed registries here - I think you can only deploy images in your GCP project's registries (i.e., I don't think you can deploy public images on Dockerhub or in other projects).